peripherals/syscfg/syscfg_f303_3x8.yaml

```
@@ -0,0 +1,276 @@
+SYSCFG:
+  CFGR1:
+    I2C3_FMP:
+      Standard: [0, "FM+ mode is controlled by I2C_Pxx_FMP bits only"]
+      # TODO: Different descripton to 37x / syscfg_f3.yaml
+      FMP: [1, "FM+ mode is enabled on all I2C3 pins selected through selection trhough IOPORT control registers AF selection bits"]
+    ENCODER_MODE:
+      NoRedirection: [0, "No redirection"]
+      # TODO: Naming?
+      MapTim2Tim15: [1, "TIM2 IC1 and TIM2 IC2 are connected to TIM15 IC1 and TIM15 IC2 respectively"]
+      MapTim3Tim15: [2, "TIM3 IC1 and TIM3 IC2 are connected to TIM15 IC1 and TIM15 IC2 respectively"]
+      MapTim4Tim15: [3, "TIM4 IC1 and TIM4 IC2 are connected to TIM15 IC1 and TIM15 IC2 respectively (STM32F303xB/C and STM32F358xC devices only)"]
+    DAC2_CH1_DMA_RMP:
+      # Note: In STM32F303x6/8 and STM32F328x8, this bit must be set
+      NoRemap: [0, "No remap"]
+      Remap: [1, "DAC2_CH1 DMA requests mapped on DMA1 channel 5"]
+    TIM7_DAC1_CH2_DMA_RMP:
+      # Note:    In STM32F303x6/8 and STM32F328x8, this bit must be set as there is no DMA2 in these products.
+      NoRemap: [0, "TIM7_UP and DAC_CH2 DMA requests mapped on DMA2 channel 4 in STM32F303xB/C and STM32F358xC devices"]
+      Remap: [1, "TIM7_UP and DAC_CH2 DMA requests mapped on DMA1 channel 4"]
+    # TODO: Check if this should be renamed or not. Conflicting with register map
+    TIM6_DAC1_CH1_DMA_RMP:
```

This is a case, where documentation name and the name in the register map are not the same.

As of RM0316 p. 261, this should be named:

```suggestion
    TIM6_DAC1_DMA_RMP:
```

I don't know which one to follow. When following the naming in the register map the patch [`devices/common_patches/f3_syscfg_cfgr1_timx_dacx_chx_dma_rmp.yaml`](https://github.com/stm32-rs/stm32-rs/pull/375/files#diff-d3ed5e32b8b8c96527fd5dc0597adb2b) wouldn't be necessary. 

As of now, I have a mixed case of applying patches to conform the documentation name (this example) but later still use names, that are used in the register map. IMO I should tend towards the register map names, as they also conform to the svd file and thus there is no need to patch all these names. But it would be harder to find the relevant documentation, if someone would search for it in the reference manual.

---

That is frustrating. I think it's not too important which one we follow:

* either way might cause confusion compared to the RM
* an extra patch is not a big overhead
* users (and HAL writers) will probably be checking the PAC documentation or autocompletion anyway

If you'd rather use the register map names that's fine with me.

---

I'll keep it as is.  I prefer it, when a search in the reference manual matches the flag description rather than the register map.